components/Typography/src/private/MDCTypographyUtilities.m

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCTypographyUtilities.h"
+
+#import "UIApplication+AppExtensions.h"
+
+/**
+ @return Device's current UIContentSizeCategory or UIContentSizeCategoryLarge
+ if we are unable to query the device due to being in an extension.
+ */
+UIContentSizeCategory GetCurrentSizeCategory(void) {
```

Comment:
I'm afraid there isn't a good way to test this functionality :/
There may be some crazy swizzling that would allow us to work some magic, but the chances of that offering a consistent, testable state is slim.

Reply:
I've opened #6931 to track this issue, but I'm going to merge the PR as this is a private implementation detail that we can change.